drivers/hdc1000/hdc1000.c

```diff
@@ -67,14 +68,18 @@ int hdc1000_test(hdc1000_t *dev)
     char reg[2];
     uint16_t tmp;
 
+    i2c_acquire(dev->i2c);
     if (i2c_read_regs(dev->i2c, dev->addr, HDC1000_MANUFACTURER_ID, reg, 2) != 2) {
+        i2c_release(dev->i2c);
         return -1;
     }
     tmp = ((uint16_t)reg[0] << 8) | reg[1];
     if (tmp != HDC1000_MID_VALUE) {
+        i2c_release(dev->i2c);
         return -1;
     }
     
+    i2c_release(dev->i2c);
```

Comment:
You can merge this i2c_release and the one on L78 by inserting `i2c_release(dev->i2c);` to between L75,L76